mesonbuild/modules/python.py

```diff
@@ -318,8 +319,11 @@ def links_against_libpython():
     cmd.ensure_finalized()
     return bool(cmd.get_libraries(Extension('dummy', [])))
 
+variables = sysconfig.get_config_vars()
+variables.update({'base_prefix': getattr(sys, 'base_prefix', sys.prefix)})
```

Comment:
This is incorrect, it assumes the target Python is the same as the Python Meson is running under. Why is it needed?

Reply:
I don't think it does? This is all happening inside the INTROSPECT_COMMAND which is run by the target Python.

Reply:
Ah, yes, sorry, my bad.